db_stress_tool/db_stress_test_base.cc

```diff
@@ -1718,6 +1727,7 @@ Status StressTest::TestBackupRestore(
         FLAGS_backup_max_size * 1000000 /* rate_bytes_per_sec */,
         1 /* refill_period_us */));
   }
+  backup_opts.current_temperatures_override_manifest = thread->rand.OneIn(2);
```

Comment:
Just curious why decide to not make this a flag?

Reply:
Not much interesting reason - just to align with most of the others in backup_opts